code/index.html

```diff
@@ -26,18 +26,29 @@ <h1>Does the person have</h1>
         <optgroup label="hair color">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+           <option value="hidden">hidden hair</option>
```

Comment:
A small thing, this line could match the indentation of the rest of the option tags